services/QuillLMS/app/services/demo/concept_results.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-module Demo::ConceptResults
```

Comment:
This is an interesting point.  I think that for a lot of these cases, my gut feeling is to leave these local variables and function params unchanged because the hope is that within a month or two we'll be rotating all references to `old_concept_results` back to the replacement `concept_results` records when the new model is in place.  So while I agree that there's some mental juggling to be done in the short term, it'll simplify things in the longer-term.

Reply:
(Note, I'm also trying not to touch front-end code, so as far as HTTP payloads are concerned, this is still `concept_result`.  Agree that, as with the above, this can be confusing in the short-term, but I'm hoping that the simplification in the long-term will pay off.